### PR TITLE
Bugfix for issue #1

### PR DESCRIPTION
**Pull request by ksyuzi**
test_encode in test_iot_ext_codec.py was failing because function encode in item_codec.py and function encode in metric_data_item_codec.py were using Enum names instead of str values. Issue was fixed, now the test passes.

### CODE REVIEW

File: tests/codecs/test_iot_ext_codec.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 class IoTextCodecTest(TestCase):
 
-    # TODO: verify where is a bug!?
+
     @skip
```
Comment (przemek1236) [inline tests/codecs/test_iot_ext_codec.py:39]: I think you can remove this skip safely now.

Comment (ksyuzi) [reply tests/codecs/test_iot_ext_codec.py:39]: oh I fixed only second test, let me take a look at the skipped one. I will reply a bit later.

Comment (przemek1236) [reply tests/codecs/test_iot_ext_codec.py:39]: Then I think we should keep the comment

Comment (przemek1236) [reply tests/codecs/test_iot_ext_codec.py:39]: The explanation of the issue with test_decode tests case has been described in the issue #1, you can take a look.

Comment (ksyuzi) [reply tests/codecs/test_iot_ext_codec.py]: second test also fixed.

**Review by przemek1236 — APPROVED** [review]

**Comment by bieli** [comment]
@ksyuzi Thanks for collaboration and fixing bug ;-)

**Pull request merged**